app/src/main/AndroidManifest.xml

```diff
@@ -1,109 +1,108 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:tools="http://schemas.android.com/tools"
-  package="org.oppia.app">
+  package="org.oppia.android">
   <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
   <uses-permission android:name="android.permission.INTERNET" />
   <!-- TODO(#56): Reenable landscape support. -->
-  <uses-sdk android:maxSdkVersion="28" />
   <application
-    android:name=".application.OppiaApplication"
+    android:name=".app.application.OppiaApplication"
     android:allowBackup="true"
     android:icon="@mipmap/ic_launcher"
     android:label="@string/app_name"
     android:roundIcon="@mipmap/ic_launcher_round"
     android:supportsRtl="true"
     android:theme="@style/OppiaTheme">
-    <meta-data android:name="firebase_analytics_collection_deactivated" android:value="true" />
-    <meta-data android:name="firebase_crashlytics_collection_enabled" android:value="false" />
+    <meta-data android:name="firebase_analytics_collection_deactivated" android:value="false" />
+    <meta-data android:name="firebase_crashlytics_collection_enabled" android:value="true" />
```

Comment:
Great catch--this should be disabled. I'll actually file an issue to add a test to verify this so that this doesn't get accidentally regressed in the future.

#1903 for that.

Reply:
Fixed in https://github.com/oppia/oppia-android/pull/1876/commits/5239cfe0bbf8f79c2bed7a489ab628c15ebac556.